di-ipv-ticf-stub/openAPI/ticf-external.yaml

```diff
@@ -83,3 +113,15 @@ components:
           items:
             type: string
           description: An array of VC's from CRI's
+    UserEvidenceRequestBody:
+      description: Request body for Ticf management
+      type: object
+      properties:
+        userId:
+          type: string
+          items:
+            type: string
+          description: The code for userId data
+        evidence:
+          type: string
```

Comment:
I think this is an object, rather than a string?

Reply:
done